go.mod

```diff
@@ -3,6 +3,7 @@ module sigs.k8s.io/scheduler-plugins
 go 1.17
 
 require (
+	github.com/dustin/go-humanize v1.0.0
```

Comment:
not this was already pulled in, because of the indirect dep to etcd:
```
$ go mod why github.com/dustin/go-humanize
warning: ignoring symlink /home/fromani/go/src/github.com/kubernetes-sigs/scheduler-plugins/manifests/install/charts/as-a-second-scheduler/templates/crds
# github.com/dustin/go-humanize
sigs.k8s.io/scheduler-plugins/cmd/controller/app
k8s.io/apiserver/pkg/server
k8s.io/apiserver/pkg/registry/generic
k8s.io/apiserver/pkg/storage/storagebackend/factory
k8s.io/apiserver/pkg/storage/storagebackend/factory.test
k8s.io/apiserver/pkg/storage/etcd3/testserver
go.etcd.io/etcd/server/v3/embed
go.etcd.io/etcd/server/v3/etcdserver
github.com/dustin/go-humanize
```

open to suggestions though.